app/helpers/organization_helper.rb

```diff
@@ -0,0 +1,6 @@
+module OrganizationHelper
+  def display_logo_or_name organization = nil
+    organization ||= current_organization
+    organization.logo.present? ? image_tag(organization.logo, alt: "#{organization.name} logo", class: "organization-logo") : organization.name
```

Comment:
Looks good. Maybe later we can discuss if we should default to the diaperbase logo if there is no org logo uploaded.